packages/cli/src/lib/strings.ts

```diff
@@ -248,4 +252,13 @@ the PWA:
   warnPwaFailedQuality: red('PWA Quality Criteria check failed.'),
   updateConfigUsage: 'Usage\n\n:[--jdk <path-to-jdk>] [--androidSdk <path-to-android-sdk>]' +
       '(You can insert one or both of them)',
+  jdkPathIsNotCorrect: 'The jdkPath isn\'t correct, please run the following command to update ' +
+      'it:\nbubblewrap updateConfig --jdkPath path/here, such that the folder "here" contains' +
```

Comment:
Could you change `path/here` to `<path-to-jdk>` so it's consistent with `androidSdkPathIsNotCorrect`?